diff -u1rN a/programs/winecfg/about.c b/programs/winecfg/about.c
--- a/programs/winecfg/about.c	2017-10-05 15:01:01.955355163 +0100
+++ b/programs/winecfg/about.c	2017-10-05 14:58:52.141658298 +0100
@@ -115,8 +115,7 @@
             titleFont = CreateFontW(
-                -MulDiv(24, GetDeviceCaps(hDC, LOGPIXELSY), 72),
+                -MulDiv(16, GetDeviceCaps(hDC, LOGPIXELSY), 72),
                 0, 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, tahomaW);
             SendMessageW(hWnd, WM_SETFONT, (WPARAM)titleFont, TRUE);
-            SetWindowTextA(hWnd, PACKAGE_NAME);
+            SetWindowTextA(hWnd, PACKAGE_STRING);
         }
-        SetDlgItemTextA(hDlg, IDC_ABT_PANEL_TEXT, PACKAGE_VERSION);
 
@@ -170,3 +169,2 @@
             SetTextColor((HDC)wParam, 0x0000007F);
-        case IDC_ABT_PANEL_TEXT:
         case IDC_ABT_LICENSE_TEXT:
diff -u1rN a/programs/winecfg/resource.h b/programs/winecfg/resource.h
--- a/programs/winecfg/resource.h	2017-10-05 15:01:01.955355163 +0100
+++ b/programs/winecfg/resource.h	2017-10-05 15:37:05.787133340 +0100
@@ -216,3 +216,2 @@
 #define IDC_ABT_PANEL                8434
-#define IDC_ABT_PANEL_TEXT           8435
 #define IDC_ABT_TITLE_TEXT           8436
diff -u1rN a/programs/winecfg/winecfg.rc b/programs/winecfg/winecfg.rc
--- a/programs/winecfg/winecfg.rc	2017-10-05 15:01:01.955355163 +0100
+++ b/programs/winecfg/winecfg.rc	2017-10-05 15:00:31.496956911 +0100
@@ -136,7 +136,6 @@
     CONTROL         "", IDC_ABT_PANEL, "STATIC", SS_OWNERDRAW, 0, 0, 260, 140
-    LTEXT           "",IDC_ABT_TITLE_TEXT,105,30,55,30
-    LTEXT           "",IDC_ABT_PANEL_TEXT,160,43,140,8
-    CONTROL         "",IDC_ABT_WEB_LINK,"SysLink", LWS_TRANSPARENT, 105,53,106,8
+    LTEXT           "",IDC_ABT_TITLE_TEXT,100,30,200,20
+    CONTROL         "",IDC_ABT_WEB_LINK,"SysLink", LWS_TRANSPARENT, 100,53,106,8
     LTEXT           "This program is free software; you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1 of the License, or (at your option) any later version.",
-                    IDC_ABT_LICENSE_TEXT,105,64,145,66
+                    IDC_ABT_LICENSE_TEXT,100,64,145,66
     GROUPBOX        "Windows registration information", IDC_STATIC, 15, 155, 230, 55
