..7a1a77b6765801b1375491602661d466ec2fee0d

From 7a1a77b6765801b1375491602661d466ec2fee0d Mon Sep 17 00:00:00 2001
From: Jacek Caban <jacek@codeweavers.com>
Date: Sun, 17 May 2020 01:02:36 +0200
Subject: [PATCH] makedep: Install also generated typelib for installed idl
 files.

Signed-off-by: Jacek Caban <jacek@codeweavers.com>
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
---
 tools/makedep.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/makedep.c b/tools/makedep.c
index cde07f669c..653c5488e1 100644
--- a/tools/makedep.c
+++ b/tools/makedep.c
@@ -2829,6 +2829,9 @@ static void output_source_idl( struct makefile *make, struct incl_file *source,
         if (source->file->flags & FLAG_IDL_HEADER)
             add_install_rule( make, source->name, strmake( "%s.h", obj ),
                               strmake( "d$(includedir)/wine/%s.h", get_include_install_path( obj ) ));
+        if (source->file->flags & FLAG_IDL_TYPELIB)
+            add_install_rule( make, source->name, strmake( "%s.tlb", obj ),
+                              strmake( "d$(includedir)/wine/%s.tlb", get_include_install_path( obj ) ));
     }
     if (!targets.count) return;
 
-- 
2.20.1

